gpii/node_modules/flatMatchMaker/test/FlatMatchMakerTests.js

```diff
@@ -264,7 +261,7 @@
     gpii.flatMatchMaker.tests.disposeStrategy = function () {
         fluid.each(disposeStrategyFixtures, function (fixture) {
             var result = gpii.flatMatchMaker.disposeStrategy(fixture.leavesArg, fixture.solRecsArg);
-            var expected = $.extend(true, {}, fixture.solRecsArg, fixture.expect);
+            var expected = fluid.extend(true, {}, fixture.solRecsArg, fixture.expect);
```

Comment:
Ok, done. It isn't implemented yet (except as a passthrough) but best to prepare for it. The idea is to fix it the way we did with all the core structural recursion primitives in the framework (fluid.merge, fluid.copy, etc) so that they bail out after a maximum recursion depth - since stack overruns in $.extend on circular objects are an infuriating thing to debug